src/V1/Api/Dumper.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neighborhoods\Buphalo\V1\Api;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Yaml;
+use Symfony\Component\Serializer;
+
+class Dumper implements DumperInterface
+{
+    private const DUMP_FORMAT = Serializer\Encoder\YamlEncoder::FORMAT;
+    private const YAML_INDENTATION = 2;
+
+    /** @var Serializer\Serializer */
+    private $serializer;
+
+    /** @var Filesystem */
+    private $filesystem;
+
+    public function __construct()
+    {
+        // If using PHP 7.4+ style setters, we could specify an ObjectNormalizer here
+        // This would allow us to pass `AbstractOjbectNormalizer::SKIP_NULL_VALUES = true` below.
+        // However, that Normalizer actually detects and uses getters, but doesn't know about hassers,
+        // as such, our 7.2 style getters throw an exception.
+        $normalizers = [
+            // This lets us take advantage of every object's jsonSerialize() method
+            new Serializer\Normalizer\JsonSerializableNormalizer(),
+        ];
+
+        $encoders = [
+            new Serializer\Encoder\YamlEncoder(
+                new Yaml\Dumper(self::YAML_INDENTATION) // This is per-level indentation
```

Comment:
This comment seems like it should be by the const declaration

Reply:
I put it there because that's what the parameter is for and to differentiate it from the config option in serialize.